### PR TITLE
Make sure Window.dpr still has a setter

### PR DESCRIPTION
**Pull request by jonahwilliams**
This is required by the web engine implementation

### CODE REVIEW

**Comment by Hixie** [comment]
why does it have a setter?

**Comment by jonahwilliams** [comment]
It's currently used in the dom renderer of the web engine implementation.

Per my logic in the interface checker, for the intermediary period the web ui implementation is allowed to have additional methods over the native ui implementation. We should tighten this in the future such that they are identical APIs

**Comment by yjbanov** [comment]
@Hixie we working on cleaning this up, but we can't keep up with Jonah :)

File: lib/stub_ui/lib/src/ui/window.dart [hunk 1]
```diff
@@ -538,7 +538,11 @@ class Window {
   ///
   ///  * [WidgetsBindingObserver], for a mechanism at the widgets layer to
   ///    observe when this value changes.
-  double get devicePixelRatio => 1.0;
+  double get devicePixelRatio => _devicePixelRatio;
+  double _devicePixelRatio = 1.0;
+  set devicePixelRatio(double value) {
```
Comment (yjbanov) [inline lib/stub_ui/lib/src/ui/window.dart]: Let's leave a TODO to remove this. Feel free to leave `TODO(yjbanov)`

**Comment by yjbanov** [comment]
To be clear, this only affects the Web version of `dart:ui` (notice that the change is under `lib/stub_ui`). The native version is unaffected.

**Comment by Hixie** [comment]
got it :-)

**Pull request merged**